api/v1alpha1/instaslice_types.go

```diff
@@ -21,15 +21,29 @@ import (
 )
 
 type Mig struct {
-	Placements     []Placement `json:"placements,omitempty"`
-	Profile        string      `json:"profile,omitempty"`
-	Giprofileid    int         `json:"giprofileid"`
-	CIProfileID    int         `json:"ciProfileid"`
-	CIEngProfileID int         `json:"ciengprofileid"`
+	// placements provide vendor profile indexes and sizes
+	// +required
+	Placements []Placement `json:"placements"`
+	// profile provides name of the vendor profile
+	// +required
+	Profile string `json:"profile"`
+	// giprofileid provides gpu instance id of a profile
+	// +required
+	Giprofileid int `json:"giprofileid"`
```

Comment:
Case doesnt' make much sense here..

- Giprofileid
- CIProfileID
- CIEngProfileID

Reply:
Thanks, working on it.